app/Notifications/BillHasProgressed.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Group;
+use App\Models\LegiScan\Bill;
+use App\Models\LegiScan\BillHistory;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class BillHasProgressed extends Notification
+{
+    use Queueable;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(public BillHistory $bill_history) {}
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        $bill_number = $this->bill_history->bill->number;
```

Comment:
These variables are all the wrong capitalization.

Reply:
Also, what value does the work `bill_` / `bill_history_` provide to all 3 of these? Were we going to get confused in the next 6 lines about what kind of number, action, and title we were talking about? Does the scroll wheel on your mouse only go one way so you can't check back a few lines if you forget? This is paranoid naming for no reason.

Reply:
Fixed!